delivery/check-issue-exists/action.yml

```diff
@@ -28,12 +34,18 @@ runs:
             }
             `,
             { issueId: githubIssueId}
-          )
+          ).catch(error => {
+            core.info(`Error "${error.message}" while trying to find the issue with ID: ${githubIssueId}`);          
+          })
           
           if (githubIssueNode === undefined) {
             core.info(`Unable to find an issue with ID: ${githubIssueId}`)
```

Comment:
is it expected ? if not, can it be a  `core.warn`?

Reply:
This is also to make sure there are no warnings or errors if the issue is not found. We're only trying to check if it exists.